MekHQ/src/mekhq/gui/panes/CampaignOptionsPane.java

```diff
@@ -9478,6 +9478,9 @@ public void updateOptions() {
             // region Markets Tab
             // Personnel Market
             options.setPersonnelMarketName(comboPersonnelMarketType.getSelectedItem());
+            if (comboPersonnelMarketType.getSelectedItem().equals("Campaign Ops")) {
```

Comment:
Can we add a compatibility handler to the marshaller method of CampaignOptions.java , so that this will sanitize itself when users load into the client. We ask users to enter the client, save and exit, when changing versions so like to include sanitizing code like this.